services/src/api/model/datatypes.rs

```diff
@@ -1561,6 +1570,75 @@ impl From<geoengine_datatypes::operations::image::Colorizer> for Colorizer {
     }
 }
 
+impl From<Colorizer> for geoengine_datatypes::operations::image::Colorizer {
```

Comment:
i know it is not part of this PR but why are the models of colorizers different in API and datatypes?

Reply:
Chris already harmonized the nesting/flatten difference. I guess the API datatype uses types for the variants is because of OpenAPI ToSchema trait. And that the datatypes one uses inline type or how it's called is because it came before?! Is this a problem or can we keep it this way?